RUNBOOK.md

```diff
@@ -5,6 +5,51 @@
+| Length of Work Queue                   | increase(workqueue_unfinished_work_seconds{}[10m])                                                        | Done                          |
+| App Reconciliation Work Duration       | sum(increase(workqueue_work_duration_seconds_bucket{name="app_reconciliation_queue"}[10m])) by (le)       | Needs Update to use Histogram |
+| App Operation Processing Work Duration | sum(increase(workqueue_work_duration_seconds_bucket{name="app_operation_processing_queue"}[10m])) by (le) | Needs Update to use Histogram |
+
+### Usage Metrics
+
+These metrics are being used to track usage of the ArgoCD instance
+
+| Usage Metric              | Query                                                                | Status   |
+|---------------------------|----------------------------------------------------------------------|----------|
+| Number of Managed Servers | count(count by (dest_server) (argocd_app_info{project=~"$Project"})) | Done     |
+| Number of Managed Apps    | count((argocd_app_info{project=~"$Project"}))                        | Done     |
+| Number of Source Repos    | count(count by (repo) (argocd_app_info{project=~"$Project"}))        | Done     |
+| Rate of Change            | To Be Implemented                                                    | Not Done |
+
+### Service Level Indicators for App Owners
+
+These metrics are possible SLIs for the App Owners
+
+| SLI               | Query                                                                                                          | Status |
+|-------------------|----------------------------------------------------------------------------------------------------------------|--------|
+| App Sync Failures | ceil(increase(argocd_app_sync_total{dest_server=~"\$Server", project=\~"$Project", phase=~"Error\|Failed"}[10m])) | Done   |
+
+### Possible Service Level Indicators
+
+These metrics are interesting to look at but their usefulness has not yet been determined.
+
+| Metric                             | Query                                                                                                                                                                                                                 | Possible Usage | Status   |
+|------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------|----------|
+| Percentage of Apps in Sync         | sum(argocd_app_info{sync_status="Synced", dest_server=~"\$Server", dest_namespace=~"\$Namespace", project=\~"\$Project"})/sum(argocd_app_info{dest_server=~"\$Server", dest_namespace=~"\$Namespace", project=~"\$Project"}) | App Owner SLI  | Done     |
```

Comment:
This is being rendered in GitHub with strikethroughs in some words.

Reply:
Weird, my local preview didn't have those.. Fixed now.